direct_home_api/test/direct_home_api_web/controllers/health_controller_test.ex

```diff
@@ -0,0 +1,8 @@
+defmodule DirectHomeApiWeb.HealthControllerTest do
```

Comment:
All is perfect but this file should be .exs not .ex, the files .exs are files scripting and the test run like a script, we don't need compile a test, let me know if I not is clear the explication

Reply:
it is very clear, sorry about that.. I will change it